packages/suite/src/services/suite/metadata/DropboxProvider.ts

```diff
@@ -172,6 +172,26 @@ class DropboxProvider extends AbstractMetadataProvider {
         }
     }
 
+    async getMetadataFilesList() {
```

Comment:
there is already `getFileContent` `setFileContent`, so using the same logic I would name this member `getFilesList` and omit the metadata part

Reply:
1d102577f